src/senaite/patient/content/patient.py

```diff
@@ -314,18 +314,24 @@ def validate_mrn(data):
                 # nothing changed
                 return
 
-        if not patient_api.is_patient_required():
-            # MRN is not a required field
-            return
+        empty = not data.mrn
```

Comment:
This could be a better test here: `len(data.mrn) == 0`
But the above seems to be also ok, even if "0" as MRN comes in.